swagger-spec/swagger.yaml

```diff
@@ -844,6 +844,12 @@ tags:
     x-traitTag: true
     description: >-
 
+      #### 2.8
+
+
+      + Wiki relationship not shown in node detail when the wiki is disabled for that project/component.
```

Comment:
The change also affects the node list endpoint, right? So how about "/data/relationships/wikis is not present in node data when node's wiki is disabled"?